.github/workflows/main.yml

```diff
@@ -0,0 +1,12 @@
+name: Luacheck
+on: [push]
+jobs: 
+  Luacheck_Tests:
```

Comment:
For the next PR, you can add another job, say `unit_tests`, with a step in which you run the testing script.

Reply:
I'll try it out, should be able to get it in by Wednesday.

Reply:
Are you set on luaunit? [Busted](http://olivinelabs.com/busted/) seems to have much better documentation, and the AwesomeWM team said it's what they use in #527. It would be nice to see how they've structured unit testing.

Reply:
Busted is fine. Take inspiration from AwesomeWM.

Reply:
What exactly do you need tested? I haven't done unit testing before and I'm not sure it's being done correctly.

Reply:
We need to launch an Awesome instance in which we load all the widgets (with all possible configurations) to see if they work. This test suite would be called for each commit to verify if everything is OK. I know this is a lot of work, but you can start by launching Awesome as mentioned [here](https://github.com/lcpz/lain/issues/527#issuecomment-1152897552=). We can incrementally add the widget tests.